README.md

```diff
@@ -5,7 +5,6 @@ Ideas should be thoroughly discussed on the [swift-evolution forums](https://swi
 
 This repository tracks the ongoing evolution of Swift. It contains:
 
-* The [status page](https://apple.github.io/swift-evolution/), tracking proposals to change Swift.
```

Comment:
Instead of deleting this link, let's update it to point at the new URL for the status page.